submit_and_compare/mixins/scenario.py

```diff
@@ -66,7 +67,7 @@ def workbench_scenarios(cls):
         """
         module = cls.__module__
         module = module.split('.', maxsplit=1)[0]
-        directory = pkg_resources.resource_filename(module, 'scenarios')
+        directory = importlib.resources.files(module) / 'scenarios'
```

Comment:
We are performing different file operations in this python module. We should update all of them as per following migration guide to keep the code optimized.

https://importlib-resources.readthedocs.io/en/latest/migration.html